components/ui/Feed/FeedView.tsx

```diff
@@ -238,27 +152,6 @@ function FeedView({ feed, community = false, header }: FeedViewProps) {
     />
   );
 
-  const footer = () => {
```

Comment:
This is a big no-no in react. This _defines_ the functional component `footer` on every single render pass. As a result, we're always changing the definition of the component, and thus React can't optimize it.

I moved it into `Footer.tsx`, below.